laplace/subnetlaplace.py

```diff
@@ -99,11 +99,8 @@ def __init__(
             backend_kwargs=backend_kwargs,
         )
 
-        if backend is not None:
-            if not isinstance(backend, GGNInterface) and not isinstance(
-                backend, EFInterface
-            ):
-                raise ValueError("SubnetLaplace can only be used with GGN and EF.")
+        if backend is not None and not issubclass(backend, (GGNInterface, EFInterface)):
```

Comment:
Use `isinstance` instead so the vanilla GGN/EF interfaces can be used or are they considered subclasses of themselves?

Reply:
The latter is already true: https://docs.python.org/3/library/functions.html#issubclass